src/components/Preferences/Availability.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+    Box,
+    Checkbox,
+    HStack,
+    Flex,
+    Select,
+    Stack,
+    Text,
+} from "@chakra-ui/react";
+import { useState } from "react";
+
+const Availability = () => {
+    const [isReliefDisabled, setIsReliefDisabled] = useState(true);
+    const [isSabbaticalDisabled, setIsSabbaticalDisabled] = useState(true);
+
+    return (
+        <Box>
+            <Stack spacing={5} direction="column">
+                <HStack spacing={5}>
+                    <Text>Please check desired semester(s) off:</Text>
+                    <Checkbox> Fall </Checkbox>
+                    <Checkbox> Spring </Checkbox>
+                    <Checkbox> Summer </Checkbox>
+                </HStack>
+                <Stack spacing={1} direction="column">
+                    <Checkbox
+                        onChange={() => setIsReliefDisabled(!isReliefDisabled)}
+                    >
+                        {" "}
+                        I am taking relief{" "}
+                    </Checkbox>
+                    <HStack>
+                        <Text minW="450px">
+                            If yes, enter how many courses you will be teaching
+                            this year:
+                        </Text>
+                        <Select
+                            placeholder="5 Courses"
```

Comment:
"5 Courses" has no associated value
well, the value is `''`

Reply:
removed placeholder